docs/examples/bitops2-terraform-blog/README.md

```diff
@@ -0,0 +1,17 @@
+# Bitops Operations Repo
+
+Welcome to Bitops! This serves as a starting point for deploying your application to the cloud.
+
+This repo can be run as is with
```

Comment:
Looks similar to https://github.com/bitovi/bitops/tree/main/docs/examples/bitops-deploy-heyemoji/README.md

Could we add some specifics what's going on in this example and how it's different from others?
https://github.com/bitovi/bitops/tree/main/docs/examples/terraform-and-ansible

Reply:
We can update it after the blog post goes live with the URL to the blog post.  This is the example for the blog post.  I need to include the link to this repo BEFORE the blog post goes live.  Until the blog goes live we won't have the URL.  See the issue?  I did add a place holder though.

Reply:
I think having a very small brief description of what this example does would be sufficient without failing into the chicken vs egg problem with URL.

Just added a specific code suggestion here: https://github.com/bitovi/bitops/pull/299#discussion_r966404418